go.mod

```diff
@@ -127,6 +128,11 @@ require (
 	sigs.k8s.io/kustomize/kustomize/v5 v5.6.0
 )
 
+require (
```

Comment:
Nit, for consistency reasons, please put these new indirect imports down below into the existing block with indirect imports.